tests/integ/test_simplifier_suite.py

```diff
@@ -205,6 +206,38 @@ def get_max_nesting_depth(query):
     )
 
 
+def test_in_with_subquery_multiple_query(session):
+    # multiple queries
+    original_threshold = analyzer.ARRAY_BIND_THRESHOLD
+    try:
+        analyzer.ARRAY_BIND_THRESHOLD = 2
+        df0 = session.create_dataframe([[1], [2], [5], [7]], schema=["a"])
+        df = session.create_dataframe(
```

Comment:
please search for SNOW-1678419 in the code base to enable the test that is currently disabled

Reply:
thanks. added